### PR TITLE
Fix: php-cs config file 

### PR DESCRIPTION
**Pull request by TomKlotzPro**
| Q                 | A
| ----------------- | ----------
| Bug fix?          | no
| New feature?      | no    
| BC breaks?        | no     
| Related Issue     | [#566](https://github.com/algolia/algoliasearch-client-php/issues/566)  
| Need Doc update   |no

## Describe your change
This pull request is related to this issue [#566](https://github.com/algolia/algoliasearch-client-php/issues/566) 

## What problem is this fixing?

It allow php-cs-fixer to fix unordered class, interfact, methods, imports.

### CODE REVIEW

File: src/Http/Psr7/functions.php [hunk 1]
```diff
@@ -51,6 +54,8 @@ function stream_for($resource = '', array $options = array())
 
 /**
  * @internal
+ *
+ * @param mixed $maxLen
```
[the remark this answers is not in the document]
Comment (TomKlotzPro) [reply src/Http/Psr7/functions.php]: I'm just applying php-cs-fixer on this pull request I prefer to correct the doc in another one !

File: .php_cs.dist [hunk 2]
```diff
@@ -6,18 +6,31 @@ $finder = \PhpCsFixer\Finder::create()
...
+            ->setRules($rulesPhp53)
+            ->setFinder($finder);
+        $rules['pre_increment'] = false;
+    } else {
+        $rules = array(
+            '@Symfony' => true,
+            '@PhpCsFixer' => true,
+            'php_unit_test_class_requires_covers' => false,
+            'multiline_whitespace_before_semicolons' => true,
+            'array_syntax' => array('syntax' => 'long'),
```
Comment (chloelbn) [inline .php_cs.dist]: you can put it short there maybe ?

File: .php_cs.dist [hunk 3]
```diff
@@ -6,18 +6,31 @@ $finder = \PhpCsFixer\Finder::create()
     ->in(__DIR__.DIRECTORY_SEPARATOR.'bin')
     ->append(array('.php_cs.dist'));
 
-$rules = array(
-    '@Symfony' => true,
-    'array_syntax' => array('syntax' => 'long'),
-);
+    if (PHP_VERSION_ID < 50400) {
+        $rulesPhp53 = array(
```
Comment (chloelbn) [inline .php_cs.dist]: called them `$rules` here so you don't have to duplicate the `\PhpCsFixer\Config::create()`

**Comment by TomKlotzPro** [comment]
Thanks to this PR, travis teston this PR : [#560](https://github.com/algolia/algoliasearch-client-php/pull/560) will go green. cc @nunomaduro

**Comment by nunomaduro** [comment]
Thanks @TomKlotzPro, give me a couple days before validating this. I have other stuff on my todo list.

**Pull request closed without merging**